src/reducers/arcReducer.ts

```diff
@@ -487,17 +503,40 @@ const arcReducer = (state = initialState, action: any) => {
               },
               proposals: {
                 [proposalId]: {
-                  redemptions: (arr: string[]) => arr.filter((item) => item != redemptionsKey)
+                  redemptions: (arr: string[]) => arr.filter((item) => item != redemptionsKey),
                 }
               },
               redemptions: { $unset: [redemptionsKey] }
             });
           }
 
-          // Also update the beneficiary account and the dao
+          if (accountRedemptions) {
```

Comment:
we shouldn't merge this in if its broken, lets leave it on a separate branch so we can both look at it more

Reply:
It's currently just deleting the redemption once it's complete because I'm not passing `redemptions` and `accountRedemptions`. So it's not broken.

Reply:
Feel free to uncomment the two lines at `onRedeemEvent` if you find a way to update it correctly.